packages/account-v2/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "@deriv/account-v2",
+    "version": "2.0.0",
+    "main": "dist/account-v2/js/index.js",
+    "engines": {
+        "node": "18.x"
+    },
+    "scripts": {
+        "build": "rimraf dist && NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.js\" && npm run translate",
+        "serve": "rimraf dist && concurrently \"cross-env BUILD_MODE='serve' NODE_OPTIONS='-r ts-node/register' webpack --progress --watch --config ./webpack.config.js\" \"tsc -w --noEmit --preserveWatchOutput\"",
+        "start": "rimraf dist && npm run test && npm run serve",
+        "translate": "sh ./scripts/update-translations.sh"
+    },
+    "dependencies": {
+        "@deriv/api": "^1.0.0",
+        "@deriv/quill-design": "^1.2.12",
+        "classnames": "^2.3.2",
+        "i18n-iso-countries": "^7.7.0",
+        "i18next": "^23.7.11",
+        "moment": "^2.29.4",
+        "qrcode.react": "^3.1.0",
+        "react": "^17.0.2",
```

Comment:
why not the latest react?

Reply:
Will cause mismatch of version between main package json and the package json inside account-v2